<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PIDLoop - untitled.reg</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="main_layout" stretch="0,0">
      <item>
       <layout class="QHBoxLayout" name="controls">
        <item>
         <widget class="QPushButton" name="regulate_button">
          <property name="text">
           <string>Regulate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="hold_button">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string>Hold</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clear_button">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="minimize_button">
          <property name="text">
           <string>Minimize</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="spacer_2">
        <property name="styleSheet">
         <string notr="true">font-size: 3px;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_actions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="separator"/>
    <addaction name="open_config_action"/>
    <addaction name="save_config_action"/>
    <addaction name="boundary_action"/>
    <addaction name="dynamic_gain_action"/>
   </widget>
   <widget class="QMenu" name="menu_steps">
    <property name="title">
     <string>Steps</string>
    </property>
    <addaction name="step_100"/>
    <addaction name="step_50"/>
    <addaction name="step_10"/>
    <addaction name="step_5"/>
    <addaction name="step_1"/>
    <addaction name="step_05"/>
    <addaction name="step_01"/>
   </widget>
   <addaction name="menu_actions"/>
   <addaction name="menu_steps"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open_config_action">
   <property name="text">
    <string>Open Config</string>
   </property>
  </action>
  <action name="save_config_action">
   <property name="text">
    <string>Save Config</string>
   </property>
  </action>
  <action name="step_100">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </action>
  <action name="step_50">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </action>
  <action name="step_10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </action>
  <action name="step_5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="step_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="step_05">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0.5</string>
   </property>
  </action>
  <action name="step_01">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0.1</string>
   </property>
  </action>
  <action name="boundary_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Boundary</string>
   </property>
  </action>
  <action name="dynamic_gain_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dynamic Gain</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
